content/docs/2.8/scalers/azure-event-hub.md

```diff
@@ -40,7 +40,8 @@ triggers:
 - `checkpointStrategy` - configure the checkpoint behaviour of different Event Hub SDKs. (Values: `azureFunction`, `blobMetadata`, `goSdk`, default: `""`, Optional)
     - `azureFunction` - Suitable for Azure Functions & Azure WebJobs SDK. This is the default setting, when `blobcontainer` is not specified.
     - `blobMetadata` - For all implementations that store checkpoint information on blob metadata such as current C#, Python, Java and JavaScript Event Hub SDKs.
-    - `goSdk` - For all implementations using the [Golang SDK](https://github.com/Azure/azure-event-hubs-go)'s checkpointing, for example Dapr.
+    - `goSdk` - For all implementations using the [Golang SDK](https://github.com/Azure/azure-event-hubs-go)'s checkpointing.
+    - `dapr` - A dapr specific checkpoint strategy based on the `goSdk` checkpointing implementation but using the dapr naming scheme.
```

Comment:
I think the current information is an implementation detail for end-users, what about simply this?
```suggestion
    - `dapr` - Used for Dapr applications using Azure Event Grid.
```